### PR TITLE
OSDOCS-1839: Add release note for configurable tune.maxrewrite and tune.bufsize for HAProxy

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-1839

Preview: https://deploy-preview-34175--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-configurable-headerBufferMaxRewriteByte-headerBufferBytes-parameters

### CODE REVIEW

**Comment by bmcelvee** [comment]
@sgreene570 and @quarterpin here is the release note PR for https://github.com/openshift/openshift-docs/pull/34171. PTAL, thank you!

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -536,6 +536,13 @@ For more information, see xref:../networking/dns-operator.adoc#nw-controlling-dn
 ==== Provider networks support for clusters that run on {rh-openstack}
 {product-title} clusters on {rh-openstack-first} now support provider networks for all deployment types.
 
+[id="ocp-4-8-configurable-headerBufferMaxRewriteByte-headerBufferBytes-parameters"]
+==== Configurable tune.maxrewrite and tune.bufsize for HAProxy
+
+Cluster Administrators can now set `headerBufferMaxRewriteByte` and `headerBufferBytes` Ingress Controller parameters to configure `tune.maxrewrite `and `tune.bufsize` HAProxy memory options per-Ingress Controller.
```
Comment (sgreene570) [inline release_notes/ocp-4-8-release-notes.adoc:542]:
```suggestion
Cluster Administrators can now set `headerBufferMaxRewriteByte` and `headerBufferBytes` Ingress Controller tuning parameters to configure `tune.maxrewrite `and `tune.bufsize` HAProxy memory options per-Ingress Controller.
```

Comment (sgreene570) [reply release_notes/ocp-4-8-release-notes.adoc:542]: Might want to be explicit here since these new API fields fall under the tuning options field.

Comment (bmcelvee) [reply release_notes/ocp-4-8-release-notes.adoc]: Done! Thank you!

**Comment by sgreene570** [comment]
LGTM

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -536,6 +536,13 @@ For more information, see xref:../networking/dns-operator.adoc#nw-controlling-dn
 ==== Provider networks support for clusters that run on {rh-openstack}
 {product-title} clusters on {rh-openstack-first} now support provider networks for all deployment types.
 
+[id="ocp-4-8-configurable-headerBufferMaxRewriteByte-headerBufferBytes-parameters"]
+==== Configurable tune.maxrewrite and tune.bufsize for HAProxy
+
+Cluster Administrators can now set `headerBufferMaxRewriteByte` and `headerBufferBytes` Ingress Controller tuning parameters to configure `tune.maxrewrite `and `tune.bufsize` HAProxy memory options per-Ingress Controller.
```
Comment (mikemckiernan) [inline release_notes/ocp-4-8-release-notes.adoc:542]: might just be me, but I was thrown a little by the ID= value *not* matching the heading.

Also, maybe use two sentences?  "...Controller tuning parameters.  These parameters correspond to the...`

That sentence was a lot for this newb to take in at one time.

**Review by mikemckiernan — APPROVED** [review]
two "up to you" ideas

**Pull request merged**